libs/blocks/article-feed/article-feed.js

```diff
@@ -520,14 +526,13 @@ async function decorateFeedFilter(articleFeedEl, config) {
   parent.parentElement.insertBefore(selectedContainer, parent);
 }
 
-const clearBlock = (block) => { block.innerHTML = ''; };
-
-export default async function init(articleFeed) {
-  const config = readBlockConfig(articleFeed);
-  clearBlock(articleFeed);
+export default async function init(el) {
+  const config = readBlockConfig(el);
+  blogIndex.config = config;
+  el.innerHTML = '';
   await loadTaxonomy();
   if (config.filters) {
-    decorateFeedFilter(articleFeed, config);
+    decorateFeedFilter(el, config);
```

Comment:
Since you're now caching the config in `blogIndex.config`, is it still worth passing the `config` between multiple functions? It might be easier to just read `blogIndex.config` wherever needed and lower the number of arguments where `config` is used.

Reply:
Great point! On it.